Runtime/Plataforms/MacOSBuildPlatform.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.Build.Reporting;
+using GameWorkstore.Patterns;
+
+namespace GameWorkstore.Automation
+{
+    [CreateAssetMenu(fileName = nameof(MacOSBuildPlatform), menuName = "Automation/" + nameof(MacOSBuildPlatform))]
+    public class MacOSBuildPlatform : StandaloneBuildPlatform
+    {
+        public override void OnBuild()
+        {
+            //Version
+            if (!UnityEditorInternal.InternalEditorUtility.isHumanControllingUs)
+            {
+                Version(out var version, out _);
+                PlayerSettings.bundleVersion = version;
+            }
+
+            //SetScriptDefinitions
+            if (UseCustomScriptDefinitions)
+            {
+                PlayerSettings.SetScriptingDefineSymbols(
+                    UnityEditor.Build.NamedBuildTarget.Standalone,
+                    ScriptDefinitions.Definitions
+                );
+            }
+
+            //Backend
+            PlayerSettings.SetScriptingBackend(BuildTargetGroup.Standalone, ScriptingBackend);
+
+            //Options
+            var buildOptions = new BuildPlayerOptions
+            {
+                scenes = GetScenes(),
+                locationPathName = "Build/MacOS/" + buildScript.GameName.ToLower().Replace(" ","_") + "_" + PlayerSettings.bundleVersion.Replace(".","_") + ".app",
```

Comment:
breaking changes: new name will cause issues to people that are current using this package. Make this change a boolean option on this platform instead.